_topic_map.yml

```diff
@@ -248,13 +248,13 @@ Topics:
     File: installing-bare-metal-network-customizations
   - Name: Installing a user-provisioned bare metal cluster on a restricted network
     File: installing-restricted-networks-bare-metal
-- Name: Installing on a single node
+- Name: Installing single node OpenShift (SNO)
```

Comment:
Optional:

I'd suggest dropping the (SNO) in the title here, and would recommend one of the following instead: 

- Installing Single Node OpenShift
- Installing SNO

And then spelling out Single Node OpenShift (SNO) in the assembly/modules.